tests/test_pdf_chisquared.py

```diff
@@ -0,0 +1,26 @@
+#  Copyright (c) 2024 zfit
+import numpy as np
+import pytest
+
+import zfit
+from zfit import Parameter
+from zfit.models.dist_tfp import ChiSquared
+
+N_true = 2
+
+obs = zfit.Space(obs="Nobs", limits=(0, 200))
+
+test_values = np.random.uniform(low=0, high=100, size=100)
+
+
+def test_chisquared():
+    N = Parameter("N", N_true)
+    chi2 = ChiSquared(obs=obs, ndof=N)
+
+    probs1 = chi2.pdf(x=test_values)
+    probs1 = probs1.numpy()
+
+    samples = chi2.sample(10000).numpy()
```

Comment:
```suggestion
    samples = chi2.sample(10000)
```
`.numpy()` is not needed in general